src/js/ripple/bignumber.js

```diff
@@ -0,0 +1,28 @@
+var BigNumber = require('bignumber.js');
+var extend = require('extend');
+
+function BigNumberWrapper(value, base) {
+  // reset config every time a BigNumber is instantiated so that
```

Comment:
I think that would be overkill. The main risk is that somebody adds a new file to the project that tries to configure BigNumber.js differently at require-time.